src/client/content_cao.h

```diff
@@ -100,6 +100,9 @@ class GenericCAO : public ClientActiveObject
 	float m_animation_speed = 15.0f;
 	float m_animation_blend = 0.0f;
 	bool m_animation_loop = true;
+	// Mesh animation frame stashed when expiring the visuals.
+	// Not used for anything else.
+	s32 m_mesh_anim_frame = 0;
```

Comment:
I could have used the existing m_anim_frame variable, but it looks like that's meant for sprites, and I did not want to make things confusing.